app/controllers/payouts/payout-list.controller.js

```diff
@@ -18,9 +18,12 @@ const listAllServicesPayouts = async function listAllServicesPayouts (req, res)
       return
     }
     const payoutSearchResult = await payoutService.payouts(userPermittedAccountsSummary.gatewayAccountIds, req.user, page)
-    const logContext = { current_page: page }
+    const logContext = {
+      current_page: page,
+      internal_user: req.user.internalUser,
+      gateway_account_ids: userPermittedAccountsSummary.gatewayAccountIds
```

Comment:
could we also log the number of live services here?

Reply:
Updated!